### DISCUSSION TITLE
Incoming Call

### DISCUSSION THREAD

**Discussion by Triplex66** [Claus-E/teams2tasmota#2] [Ideas]
Hello Claus, Do you think it is Possibly you can Signal a Incoming Call with flash LED or only Staple Color?

I have the Problem that Rings Only my Headset at my Notebook.

Thanks

Best Regards 
Thomas

**Comment by nagyrobi** [comment]
+1 for flashing light on incoming call !!

**Comment by Claus-E** [comment]
Hi guys,

flashing on incoming calls is now implemented. Please use the actual xml.
The flashing colors can edited: right click on "*NotificationColor1" or "*NotificationColor2"

Hope it works, I did not have much time for testing :-)

Claus

**Reply by Triplex66** [reply]
Short test.  it works perfectly

Thank you very much

**Reply by Triplex66** [reply]
I Test some days, it work very well but the the Notification on incoming Chats cannot be disabled. Its flash by call and incoming chats.
The Tool is very helpful. Thanks again

**Reply by Claus-E** [reply]
Hi Thomas,

next try with new Version :-) 
have a lock on Settings.

by the way, wich type of tasmota-device did you use? Just for interest.

Claus

**Reply by nagyrobi** [reply]
I use https://templates.blakadder.com/athom_LB017W.html

**Reply by Triplex66** [reply]
Hi Claus, thanks for your Support.

First Test Locks Gut, I use a Diy Tasmota with 1 ESP8266-01S and 8 X WS2812.
![MicrosoftTeams-image (1)](https://user-images.githubusercontent.com/79790860/111061701-fb446a80-84a4-11eb-8f4d-04a3769fb25c.png)
![MicrosoftTeams-image](https://user-images.githubusercontent.com/79790860/111061705-fe3f5b00-84a4-11eb-95fb-e6d6597f992f.png)
![MicrosoftTeams-image (3)](https://user-images.githubusercontent.com/79790860/111061713-00a1b500-84a5-11eb-82a1-d7776313cdea.png) .The Case is a Ü-Ei :-)

**Reply by Triplex66** [reply]
Hello Claus,  the new Version works fine.
nice that it is now so flexible with the setting :-)
I get a Error wen the Setting have no COM port .

![Bildschirmfoto 2021-03-15 um 11 45 26](https://user-images.githubusercontent.com/79790860/111153361-25c02180-8592-11eb-9c79-214f84232c4d.png)
![Bildschirmfoto 2021-03-15 um 10 59 12](https://user-images.githubusercontent.com/79790860/111153378-29ec3f00-8592-11eb-9ffe-817e66ae1ab5.png)

Thanks Claus

**Reply by Claus-E** [reply]
Hi Thomas,

I have tested several times
I can not recreate the error.

But i have learned that "http://tasmota-busylight2" is a valid url.
I always typed "http://tasmota-busylight2.fritz.box"  witch is not needed.
Incredible saving :-)

please try to use the aktual *.xml

If you still have the Problem please save & post the *.xml before and (if changed) after the Error.

By the way your Ü-EI looks very good!

Greetings Claus

**Reply by Triplex66** [reply]
Hi Claus,
the new Version works perfekt for me.
Do you want a Ü-Ei version a send you for free, how can I send you a PM?
best regards
Thomas

**Reply by Claus-E** [reply]
Hi Thomas,

Nice to hear, that you that you are satisfied.
PM is not possible on GitHub.
You can contact me via temp_8932@nurfuerspam.de

Greetings Claus

**Reply by Triplex66** [reply]
Froste Ostern, Mail ist raus Claus
LG
Thomas
